example/stores/weather.ts

```diff
@@ -1,8 +1,18 @@
 import { Store } from '../../src';
 import immer from 'immer';
+import { Stores } from '../synaptik';
 
-export default class WeatherStore extends Store {
-  state = {
+type Forecast = {
+  date: string;
+  high: number;
+  low: number;
+};
+
+export default class WeatherStore extends Store<WeatherStore, Stores> {
```

Comment:
Looks like theres a circular dependency.  `Stores` is generated from `{ counter, todos, weather };` and weather is generated using  `Stores`.  Code seems to work but are we missing gotcha somewhere?

Reply:
I haven't found a way around it. We need all the stores to properly use the `Synapse<T>` generic type.